Source/animationinfos.h

```diff
@@ -41,6 +56,18 @@ class AnimationInfos {
 	 */
 	int GetFrameToUseForRendering();
 
+	/**
+	 * @brief Sets the new Animation with all relevant information for rendering
+
```

Comment:
Empty line without the * at the front seems non-standard to me.

Reply:
Fixed